src/fosslight_util/oss_item.py

```diff
@@ -78,41 +88,84 @@ def set_files(self, value):
     def set_source(self, value):
         self.source = value
 
+    # new keys for updated yaml format
+    # replace 'file' key
+    def set_source_name_or_path(self, value):
+        if not isinstance(value, list):
+            value = value.split(",")
+        self.source_name_or_path.extend(value)
+        self.source_name_or_path = [item.strip() for item in self.source_name_or_path]
+        self.source_name_or_path = list(set(self.source_name_or_path))
+
+    # replace 'source' key
+    def set_download_location(self, value):
+        self.download_location = value
+
+    # replace 'copyright' key
+    def set_copyright_text(self, value):
+        self.copyright_text = value
+
+    def set_sheet_item(self, item):
+        switcher = {
+            0: self.set_source_name_or_path,
+            1: self.set_name,
+            2: self.set_version,
+            3: self.set_licenses,
+            4: self.set_download_location,
+            5: self.set_homepage,
+            6: self.set_copyright_text,
+            7: self.set_exclude,
+            8: self.set_comment
+        }
+
+        for i in range(0, len(switcher)):
+            func = switcher.get(i, "invalid")
+            func(item[i])
+
     def get_print_array(self):
         items = []
         if len(self.files) == 0:
             self.files.append("")
         if len(self.licenses) == 0:
             self.licenses.append("")
+
+        exclude = ""
+        if self.exclude:
+            exclude = "Exclude"
```

Comment:
exclude = "Exclude" if self.exclude else ""

Reply:
Changed it